ui/imports/shared/status/StatusStickerList.qml

```diff
@@ -18,8 +20,8 @@ GridView {
     clip: true
     signal stickerClicked(string hash, int packId)
     delegate: Item {
-        width: stickerGrid.cellWidth
-        height: stickerGrid.cellHeight
```

Comment:
This looks like this was referring to `cellWidth` and `cellHeight` from a different component.

Now this been changed to `root.cellWidth`/`root.cellHeight` respectively.
I don't see us passing those values to `StatusStickerList` in this PR.

So this means either: `width` and `height` of the item here was always broken, or these props potentially behave differently now.

What happened to the `stickerGrid` where those values came from?

(BTW: I see that `root` here has its own `cellWidth` and `cellheight`, but those haven't been introduced in this PR, so I want to make sure this change doesn't cause funny side effects)

Reply:
So I see 3 usages of `StatusStickerList`. One is in `StatusStickerMarket.qml` where there is no such component 'id'ed' `stickerGrid` , the other one is in `StatusStickerPackClickPopup.qml` where the component with id `stickerGrid` is an `StatusStickerPackDetails` which is basically an `Item` and has no properties `cellWidth` and `cellheight` and the last place is in `StatusStickersPopup.qml` where the component that has id `stickerGrid` is the `StatusStickerList.qml` itself which is basically a `GridView` and has `cellWidth` and `cellheight` built-in properties set to 88 pixels each.  After this analysis I assumed that this binding wasn't making much sense and the at the end the desired binding was `root.cellWidth` / `root.cellHeight` instead of  `gridView.cellWidth` / `gridView.cellHeight`. Do you think I'm missing something?

Reply:
This sounds reasonable, thanks for clarifying. If this works the same as before, this LGTM